src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Figures {
+    Square,
+    Rectangle,
+    RightTriangle,
+    Circle,
+    IsoscelesTrapezoid

```

Comment:
According to the checklist, enum constants should be written in uppercase letters. Please, correct the naming of the enum constants to follow the Java naming conventions.